mne/io/curry/curry.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: UTF-8 -*-
+#
+# Authors: Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+#
+# License: BSD (3-clause)
+
+
+import numpy as np
+import mne
+import os
+import re
+
+
+def _read_curry(full_filename):
+    '''
+    Read curry into python.
```

Comment:
we know this is python and that we are reading curry ;) no need to put it in a comment

Reply:
Yes, sorry it was just meant as a placeholder. :D